src/clients/QBittorrent.ts

```diff
@@ -234,11 +234,9 @@ export default class QBittorrent implements TorrentClient {
 	> {
 		let torrentInfo: TorrentConfiguration;
 		try {
-			if (await this.isInfoHashInClient(searchee.infoHash!)) {
-				torrentInfo = await this.getTorrentConfiguration(searchee);
-				if (torrentInfo.save_path === undefined) {
-					return resultOfErr("NOT_FOUND");
-				}
+			torrentInfo = await this.getTorrentConfiguration(searchee);
+			if (torrentInfo.save_path === undefined) {
+				return resultOfErr("NOT_FOUND");
```

Comment:
so was the isInfoHashInClient call redundant here? (looks it otherwise would blow up on the `return resultOf(torrentInfo!.save_path)`? 

can we move that return line inside the try/catch?

Reply:
@ShanaryS pointed out that the logic in `getTorrentConfiguration()` pulls the infohash, and throws if it isn't in the client, which would provide the same logic with fewer API queries to qBittorrent.